go.mod

```diff
@@ -18,17 +18,22 @@ require (
 	gopkg.in/errgo.v1 v1.0.1
 )
 
+require github.com/stretchr/testify v1.8.2
```

Comment:
This should be included in the previous `require` block

Reply:
Oops, don't know why it has been added here.
I'm moving it.